std/typecons.d

```diff
@@ -3441,28 +3441,29 @@ if (is (typeof(nullValue) == T))
     assert(ntts.to!string() == "2.5");
 }
 
+// apply
 /**
 Unpacks the content of a $(D Nullable), performs an operation and packs it again. Does nothing if isNull.
 
-When called on a $(D Nullable), $(D apply) will unpack the value contained in the $(D Nullable),
-pass it to the function you provide and wrap the result in another $(D Nullable) (if necessary).
-If the Nullable is null, $(D apply) will return null itself.
+When called on a `Nullable`, `apply` will unpack the value contained in the `Nullable`,
```

Comment:
I think we should continue the automation alternative here. At the moment we only converted std.algorithm automatically